partiql-ast/src/ast.rs

```diff
@@ -832,6 +835,164 @@ pub enum JoinSpec {
     Natural,
 }
 
+/// `<expr> MATCH <graph_pattern>`
+#[derive(Visit, Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct GraphMatch {
```

Comment:
probably need a changelog entry at some point for GPML inclusions such as this AST + parser addition

Reply:
Yes. I Intend to add the changelog once everything is hooked up.

Reply:
See https://github.com/partiql/partiql-lang-rust/issues/550